pipefy.js

```diff
@@ -50,17 +55,38 @@
         if (this.notEnoughInfo(data)) {
           return;
         }
-        var result;
 
+        this.tryBuildTimer(data);
+      };
+
+      PipefyProfile.prototype.tryBuildTimer = function(data) {
         setTimeout((function(_this) {
           return function() {
+            _this.renderTries++;
+            !debug || console.info("trying to add button");
+
+            var hasTimer = !!document.querySelector(".harvest-timer");
+            var hasActions = !!document.querySelector(_this.actionSelector);
```

Comment:
Sorry, I really don't like those `!!` to get a boolean value from a query. Isn't better just a `!= nil`?